[2024-10-03T12:20:11.590+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-10-03T12:20:11.604+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: data_preparation.split_data scheduled__2024-10-03T12:15:00+00:00 [queued]>
[2024-10-03T12:20:11.608+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: data_preparation.split_data scheduled__2024-10-03T12:15:00+00:00 [queued]>
[2024-10-03T12:20:11.608+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 1
[2024-10-03T12:20:11.617+0000] {taskinstance.py:2888} INFO - Executing <Task(_PythonDecoratedOperator): split_data> on 2024-10-03 12:15:00+00:00
[2024-10-03T12:20:11.643+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=10048) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-10-03T12:20:11.644+0000] {standard_task_runner.py:72} INFO - Started process 10057 to run task
[2024-10-03T12:20:11.645+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'data_preparation', 'split_data', 'scheduled__2024-10-03T12:15:00+00:00', '--job-id', '316', '--raw', '--subdir', 'DAGS_FOLDER/prepare_data.py', '--cfg-path', '/tmp/tmpa9co8mzj']
[2024-10-03T12:20:11.647+0000] {standard_task_runner.py:105} INFO - Job 316: Subtask split_data
[2024-10-03T12:20:11.692+0000] {task_command.py:467} INFO - Running <TaskInstance: data_preparation.split_data scheduled__2024-10-03T12:15:00+00:00 [running]> on host 8e64ac78d004
[2024-10-03T12:20:11.744+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='data_preparation' AIRFLOW_CTX_TASK_ID='split_data' AIRFLOW_CTX_EXECUTION_DATE='2024-10-03T12:15:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-10-03T12:15:00+00:00'
[2024-10-03T12:20:11.746+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-10-03T12:20:11.824+0000] {python.py:240} INFO - Done. Returned value was: None
[2024-10-03T12:20:11.833+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-10-03T12:20:11.833+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=data_preparation, task_id=split_data, run_id=scheduled__2024-10-03T12:15:00+00:00, execution_date=20241003T121500, start_date=20241003T122011, end_date=20241003T122011
[2024-10-03T12:20:11.876+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-10-03T12:20:11.885+0000] {taskinstance.py:3900} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-10-03T12:20:11.886+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
